pulp_rpm/app/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .comps import PackageCategory, PackageEnvironment, PackageGroup, PackageLangpacks  # noqa
+from .custom_metadata import RepoMetadataFile  # noqa
+from .dist_tree import Addon, Checksum, DistributionTree, Image, Variant  # noqa
+from .modulemd import Modulemd, ModulemdDefaults  # noqa
+from .package import Package  # noqa
+from .rpm import RpmDistribution, RpmPublication, RpmRemote, RpmRepository  # noqa
```

Comment:
rpm.py seems to ambiguous, i would leave this in models.py or maybe rename this to repository.py ( however it does no fully reflect all the models, but at least they are all repo related)

Reply:
+1 to repository.py

There is no longer models.py, there is only models directory now.